components/menu/menu-item/menu-item.tsx

```diff
@@ -4,16 +4,16 @@ import Link from 'next/link';
 import './menu-item.scss';
 
 type Props = {
-  href: string,
-  children: ReactNode
+  href: string;
+  children: ReactNode;
 };
 
 const MenuItem = (props: Props) => (
-  <Link href={props.href}>
-    <a className="menu-item">
-      <li>{props.children}</li>
-    </a>
-  </Link>
+  <li className="menu-item">
+    <Link href={props.href}>
```

Comment:
This is better semantically and accessibility :information_desk_person:

Reply:
Yep, this is better. Might just need to fix up the hover styles though
<img width="305" alt="Screen Shot 2020-02-23 at 10 08 08 pm" src="https://user-images.githubusercontent.com/19167953/75110998-0f92ab00-5689-11ea-9373-6a18fbc7925e.png">